layouts/_default/baseof.html

```diff
@@ -26,7 +26,7 @@
                         <span class="icon-bar"></span>
                         <span class="icon-bar"></span>
                     </button>
-                    <img class="navbar-brand" src="https://cdn.earthdata.nasa.gov/eui/latest/docs/assets/ed-logos/meatball_hover.png" alt="Meatball" />
```

Comment:
The previous SVG image looked much better. Let's use that instead of this fixed-site PNG image.